### PR TITLE
fix: handle default for non-error responses

### PR DESCRIPTION
**Pull request by lalluanthoor**
Solves issue 1 in #4

### CODE REVIEW

**Comment by lalluanthoor** [comment]
Proposed solutions for point 2 in issue #4 as well as issue #6

File: templates/src/api/routes/___route.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ router.{{@key}}('{{../../subresource}}', (req, res, next) => {
     if (err) {
     {{#each ../responses}}
       {{#compare @key 400 operator=">="}}
-      const err_response = { status: {{@key}}, message: '{{../description}}' };
+      const err_response_{{@key}} = { status: {{@key}}, message: '{{../description}}' };
       return res.status({{@key}}).send(err_response);
       {{/compare}}
       {{#equal @key "default"}}
```
Comment (fmvilas) [inline templates/src/api/routes/___route.js]: I think we can now remove this because "default" is always 200.

**Comment by fmvilas** [comment]
Everything looks good to me. Just left a comment. Thanks for the PR @lalluanthoor.

**Comment by lalluanthoor** [comment]
@fmvilas I've added the change.

**Comment by fmvilas** [comment]
Cool! Will do the release in a few minutes. Thanks for your work!

**Comment by fmvilas** [comment]
@lalluanthoor version `0.1.0` should now be available on npm.

**Pull request merged**